engine_wrapper.py

```diff
@@ -14,6 +14,7 @@ def create_engine(config):
     engine_path = os.path.join(cfg["dir"], cfg["name"])
     engine_type = cfg.get("protocol")
     engine_options = cfg.get("engine_options")
+    draw_or_resign = cfg.get("draw_or_resign", {})
```

Comment:
This should be written this way: `draw_or_resign = cfg.get("draw_or_resign") or {}`. If the user's config.yml has the following:

    draw_or_resign:
      # resign_enabled: False
      # ... etc.

Then `cfg.get("draw_or_resign", {})` will return `None` since the key `draw_or_resign` is present. Since dict methods are called on `draw_or_resign` later, these will causes errors if that variable is not a dict.